recipes/cogdl/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "cogdl" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/THUDM/cogdl/archive/{{ version }}.tar.gz
+  sha256: 4c35bcc20e9bd7e9928c6e5a02d2ca974d42ad47255ad7dafd70cbec5da079d6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
Should also have `python >=3.6` here too

Reply:
Done.